tox.ini

```diff
@@ -0,0 +1,68 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+envlist = py27,py36,flake8
+skip_missing_interpreters = False
+
+[travis]
+python =
+  3.6: py36, flake8
+
+[flake8]
```

Comment:
should we also include stuff for the import statements? e.g. https://github.com/aws/sagemaker-containers/blob/master/.flake8

Reply:
I will do that.